lms/djangoapps/mobile_api/users/urls.py

```diff
@@ -3,9 +3,11 @@
 """
 from django.conf.urls import patterns, url
 from django.conf import settings
+from rest_framework.routers import SimpleRouter
 
 from .views import UserDetail, UserCourseEnrollmentsList, UserCourseStatus
 
+ROUTER = SimpleRouter()
```

Comment:
my bad; i was testing a few ways to change the API view, forgot to remove unused `ROUTER`.
Removed now.